Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -1745,7 +1745,7 @@ Sollten Sie den Test in der App gelöscht haben, können Sie ihn aus dem Papierk
     <string name="settings_interoperability_subtitle">Teilnehmende Länder</string>
 
     <!-- XHED: Header of interoperability information/configuration view -->
-    <string name="interoperability_configuration_title">Länderübergreifende Risiko-Ermittlung</string>
+    <string name="interoperability_configuration_title">Länder\u00ADübergreifende Risiko-Ermittlung</string>
```

Comment:
what is `\u00AD` ? and why?

Reply:
@mtwalli it's a [soft hyphen](https://www.fileformat.info/info/unicode/char/00ad/index.htm) and it "indicates word division when a word is broken at the end of a line.", according to[ this website](https://jkorpela.fi/shy.html#:~:text=The%20first%2C%20called%20the%20'hard,the%20end%20of%20a%20line.) Looking at the bug, it seems to be used correctly here?

Reply:
I think ` android:hyphenationFrequency="full"` is enough? I used it before in one of the check-ins feature and afaik it works, why here we need the soft hyphen additionally ?

Reply:
Without the soft hyphen , it is exactly the same 
<img width="541" alt="Screenshot 2022-05-19 at 10 00 26" src="https://user-images.githubusercontent.com/25054729/169243820-1e586b16-43f0-40c6-a8c8-dec0d65edb43.png">
<img width="541" alt="Screenshot 2022-05-19 at 10 01 30" src="https://user-images.githubusercontent.com/25054729/169243853-112ca25b-ebdf-438f-899a-ce0cc5447bfd.png">

Reply:
With the soft hyphen it breaks always at this suggested placement:
Länder-
übergreifende
Risikoermittlung

With NO soft hyphen it breaks always at overflowing placement:
e.g.:
Länderübergre-
ifende Risiko-
ermittlung

@KathrinLuecke what do you prefer? Soft hyphen, or NO Shoft hyphen?

Reply:
Thanks, I'm fully convinced now, removed the hyphen